specifications/active/aRMT-1.6.1.yml

```diff
@@ -0,0 +1,138 @@
+name: aRMT
```

Comment:
any reason why these two versions of the spec can't be merged into one version?

Reply:
Because this was accidentally deleted, but it was released in a previous version.